pythran/analyses/aliases.py

```diff
@@ -128,6 +128,14 @@ def visit_Set(self, node):
         self.generic_visit(node)
         return self.add(node)  # not very accurate
 
+    def visit_Return(self, node):
+        if not node.value:
+            return
+        for alias in self.visit(node.value):
+            # FIXME: handle subscript
+            if type(alias) is ast.Name and type(alias.ctx) is ast.Param:
```

Comment:
You should use "isinstance" to be consistent with other part of the code and use PEP8 recommendation.